index.html

```diff
@@ -16,12 +16,13 @@
           <main> 
             <h1>Welcome to Carlin' CSD120 Website! </h1> 
                   <p> 
-                     I am currently a student at Sault College! and I'm planning on finishing my two years of Computer Programming <br>
-                     so I could be one step closer to obtaining a Computer Science Degree. In past experiences, I have worked as I.T <br>
-                     at my local schools in my hometown for a Summer of work and 2 separate classes in Co-Op for school. <br>
-                     <br>
-                     On my freetime, I like to help out at home and be busy. My usual pass times consist of playing games, movies and reading.<br>
-                     My hometown is in Moose Factory which is way up North of Ontario and I have decided to come live in Sault Ste Marie for my <br>
+                     I am currently a student at Sault College! and I'm planning on finishing my two years of Computer Programming
+                     so I could be one step closer to obtaining a Computer Science Degree. In past experiences, I have worked as I.T
+                     at my local schools in my hometown for a Summer of work and 2 separate classes in Co-Op for school. 
+                  </p>
+                  <p>
+                     On my freetime, I like to help out at home and be busy. My usual pass times consist of playing games, movies and reading.
+                     My hometown is in Moose Factory which is way up North of Ontario and I have decided to come live in Sault Ste Marie for my
```

Comment:
Don't use `<br>` elements to manage whitespace in a paragraph (unless it's required as in the case of breaking lines of poetry below).  Use one `<p>` element per paragraph.